vulcanize/uniswap-watcher-ts
# Refactor methods in uni-info-watcher custom-indexer

Part of https://github.com/cerc-io/watcher-ts/issues/191
Requries https://github.com/cerc-io/watcher-ts/pull/238

- Use latest entity query from watcher-ts
- Use lateral query pattern from watcher-ts
- Refactor code to use methods from watcher-ts
- Use normal entity queries which check entity is_pruned flag
- Remove custom-indexer file
